### PR TITLE
Display events of non-running pods on E2E tests failure

### PR DESCRIPTION
**Pull request by sebgl**
This should help tracking problems with Pods that stay Pending in e2e
tests (lack of CPU, unbound PVs, etc.).

Get the list of pods that are not running: for each one grep the `Events` section of the `kubectl describe pod $pod` output.

### CODE REVIEW

File: operators/hack/run-e2e.sh [hunk 1]
```diff
@@ -81,6 +81,14 @@ else
     echo "--"
     kubectl -n $NAMESPACE get elastic,pods,rs,deploy,svc,cm,secrets,pvc,pv
     echo "--"
+    echo "# events for non-running pods"
+    for pod in $(kubectl get pods -n $NAMESPACE | tail -n +2 | grep -v Running | awk '{print $1}')
```
Comment (thbkrkr) [inline operators/hack/run-e2e.sh]: FWIW, a more kubectl-ish version could be:

```
kubectl -n $NAMESPACE get pods \
	--no-headers \
	--field-selector=status.phase=Running \
	-o custom-columns=:metadata.name \
| xargs -I {} kubectl -n $NAMESPACE get event \
	--field-selector involvedObject.name={}
```

**Comment by artemnikitin** [comment]
> we should probably do #556 quickly (use eck-dump.sh)

I can do it on next week (after release). Or if someone has some free time now, then feel free to take it.

File: operators/hack/run-e2e.sh (hunk 1 above)
Comment (sebgl) [reply operators/hack/run-e2e.sh]: Indeed that looks better

Comment (sebgl) [reply operators/hack/run-e2e.sh]: I'll remove the xargs part which makes it harder to chain "echo $pod && cmd", keep the for loop but use your kubectl args.

**Pull request merged**